api/src/test/java/org/openmrs/module/ModuleUtilTest.java

```diff
@@ -79,7 +82,58 @@ public void getMandatoryModules_shouldReturnMandatoryModuleIds() throws Exceptio
 		//then
 		assertThat(ModuleUtil.getMandatoryModules(), contains("firstmodule"));
 	}
-	
+
+
+	/**
+	 * @verifies return false when versions is null
+	 * @see ModuleUtil#isOpenmrsVersionInVersions(String[])
+	 */
+	@Test public void isOpenmrsVersionInVersions_shouldReturnFalseWhenVersionsIsNull() throws Exception {
+
+		Assert.assertFalse(ModuleUtil.isOpenmrsVersionInVersions(null));
+	}
+	/**
+	 * @verifies return false when versions is empty
+	 * @see ModuleUtil#isOpenmrsVersionInVersions(String[])
+	 */
+	@Test public void isOpenmrsVersionInVersions_shouldReturnFalseWhenVersionsIsEmpty() throws Exception {
+
+		Assert.assertFalse(ModuleUtil.isOpenmrsVersionInVersions( new String[] {}));
+	}
+
+	/**
+	 * @verifies return true if current openmrs version matches one element in versions
+	 * @see ModuleUtil#isOpenmrsVersionInVersions(String[])
+	 */
+	@Test public void isOpenmrsVersionInVersions_shouldReturnTrueIfCurrentOpenmrsVersionMatchesOneElementInVersions()
+			throws Exception {
+
+		Field versionField = OpenmrsConstants.class.getDeclaredField("OPENMRS_VERSION_SHORT");
+		Field modifiersField = Field.class.getDeclaredField("modifiers");
+		modifiersField.setAccessible(true);
+		modifiersField.setInt(versionField, versionField.getModifiers() & ~Modifier.FINAL);
+		final String currentVersion = "1.9.8";
+		versionField.set(null, currentVersion);
+
+		Assert.assertTrue(ModuleUtil.isOpenmrsVersionInVersions( currentVersion, "1.10.*"));
```

Comment:
Shouldn't this be false? Because 1.9.8 isn't matched by the expression **1.10.***